tests/integration/utils.py

```diff
@@ -240,12 +247,15 @@ def fetch_deps_and_check_output(
         output_json = _load_json(output_folder.joinpath("output.json"))
         expected_output_json = _load_json(test_data_dir.joinpath(test_case, "output.json"))
 
-        if "project_files" in expected_output_json:
-            _set_tmpdir_path(expected_output_json["project_files"], tmp_path)
+        output_json["project_files"] = expected_output_json["project_files"]
+        # if "project_files" in expected_output_json:
+        #     _set_tmpdir_path(expected_output_json["project_files"], tmp_path)
 
         log.info("Compare output.json files")
         assert output_json == expected_output_json
 
+        _update_test_data_if_needed(test_data_dir.joinpath(test_case, "output.json"), output_json)
```

Comment:
Maybe we could update data before loading the expected data?
```python
actual_data = ...
_update_test_data_if_needed(actual_data, expected_data_file)
expected_data = _load_json(expected_data_file)
assert actual_data == expected_data
```
This way the tests won't fail in generating mode (if the data is generated correctly)